README.md

```diff
@@ -477,6 +528,44 @@ Sent when the user interrupts the assistant, after the user has finished speakin
 
 Sent when the assistant decides to end the conversation.
 
+#### `tool_call`
+
+```ts
+{
+  type: "tool_call";
+  id: string;
+  tool: {
+    id: string;
+    name: string;
+  };
+  endpoint_url: string | null;
+  endpoint_timeout_ms: number | null;
+  endpoint_called_at: string | null;
+  request_body: object | null;
+  response_body: object | null;
+  response_status_code: number | null;
+  timed_out: boolean | null;
+  error_message: string | null;
+}
```

Comment:
@qionghuang6 going to slack you some info about a customer requirement for the tool call spec